osu.Game/Skinning/SkinnableSamples.cs

```diff
@@ -16,9 +16,10 @@
 namespace osu.Game.Skinning
 {
     /// <summary>
-    /// A sound consisting of one or more samples to be played.
+    /// A group of (generally gameplay) samples to be played together as one.
+    /// Unlike <see cref="SkinnableSample"/>, this uses <see cref="IPooledSampleProvider"/> internally to retrieve all samples from pools when available.
     /// </summary>
-    public partial class SkinnableSound : SkinReloadableDrawable, IAdjustableAudioComponent
+    public partial class SkinnableSamples : SkinReloadableDrawable, IAdjustableAudioComponent
```

Comment:
Not sure how to feel about these naming changes. I would expect the only difference between a `SkinnableSample` and a `SkinnableSamples` to be the... well, plurality of them; but it is not so, `SkinnableSamples` has a lot more stuff like using `IPooledSampleProvider` and respecting `ISamplePlaybackDisabler`, and `MinimumSampleVolume`...

I'd say the naming needs to do a better job distinguishing between the two. Maybe do something like `SkinnableSamples` -> `GameplaySamples`, as that's what it kinda is.